src/utils/pageWrappers/withDataSSR.js

```diff
@@ -50,9 +50,7 @@ const withDataSSR = (getRelayQuery) => (getServerSidePropsFunc) => async (
   }
 
   return {
-    // TODO: put in a "props" key when next-firebase-auth modifies
-    //   its composition.
-    ...composedProps,
+    props: { ...composedProps },
```

Comment:
We actually need everything in the props key, like:
```js
return {
    ...composedProps,
    props: {
      ...composedProps.props,
      // If we don't fetch data, it should be null so that SWR will
      // fetch data on the client side (in `useData`).
      data: isEmpty(queryProps)
        ? null
        : {
            ...queryProps,
          },
      // The "initialRecords" prop is consumed by the `withRelay` HOC.
      initialRecords,
    },
  }
```

As-is, the index page throws. See NFA [breaking changes](https://github.com/gladly-team/next-firebase-auth/releases/tag/v0.11.0).